source/common/http/codes.cc

```diff
@@ -18,111 +18,132 @@
 namespace Envoy {
 namespace Http {
 
-void CodeStatsImpl::chargeBasicResponseStat(Stats::Scope& scope, const std::string& prefix,
+CodeStatsImpl::CodeStatsImpl(Stats::SymbolTable& symbol_table)
+    : stat_name_pool_(symbol_table), symbol_table_(symbol_table), canary_(makeStatName("canary")),
+      canary_upstream_rq_time_(makeStatName("canary.upstream_rq_time")),
+      external_(makeStatName("external")),
+      external_rq_time_(makeStatName("external.upstream_rq_time")),
```

Comment:
This and the next line are the same. That seems not correct?

Reply:
Yes, although I think some of these StatNames were not referenced. Fixed.